src/scribe_data/cli/cli_utils.py

```diff
@@ -122,4 +122,4 @@ def print_formatted_data(data: Union[Dict, List], data_type: str) -> None:
                 print(item)
 
     else:
-        print(data)
+        print(data)
```

Comment:
For here, we do want the line at the end of the file. Should also be fixed by pre-commit :)